modules/services/images/src/main/java/au/com/shiftyjelly/pocketcasts/images/PocketCastsIcons.kt

```diff
@@ -0,0 +1,21 @@
+package au.com.shiftyjelly.pocketcasts.images
+
+import androidx.compose.ui.graphics.vector.ImageVector
+import au.com.shiftyjelly.pocketcasts.images.icons.StarEmpty
+import au.com.shiftyjelly.pocketcasts.images.icons.StarFull
+import au.com.shiftyjelly.pocketcasts.images.icons.StarHalf
+
+private var icons: List<ImageVector>? = null
+
+object PocketCastsIcons {
+    val AllIcons: List<ImageVector>
+        get() {
+            var iconsLoaded = icons
+            if (iconsLoaded != null) {
+                return iconsLoaded
+            }
+            iconsLoaded = listOf(StarFull, StarEmpty, StarHalf)
+            icons = iconsLoaded
+            return iconsLoaded
+        }
```

Comment:
This isn't used for anything yet, but SVG to Compose generated it, so I thought I should keep it for now. Let me know if you think I should delete it.

Reply:
If it isn't used, we can remove it.